### ISSUE TITLE
Compiler warning from 

### ISSUE THREAD

**Issue by BartMassey** [BartMassey/rocket-pastebin#2]
Compilation currently elicits a deprecation warning:
```
warning: `#[derive]` for custom traits is deprecated and will be removed in v1.15. Prefer using procedural macro custom derive
   --> src/main.rs:108:10
    |
108 | #[derive(FromForm)]
    |          ^^^^^^^^
```

Not much to do about this except wait for Rocket to move forward, I think.

**Comment by botandrose** [comment]
FWIW, I don't get this deprecation warning. Perhaps I have newer versions of dependencies.

**Comment by BartMassey** [comment]
Resolved by commit 6c965e1; require newer version of Rocket in `Cargo.toml`.
